Dockerfile

```diff
@@ -1,8 +1,10 @@
-FROM node:18.14.2-slim
+FROM node:18.15.0
```

Comment:
I assume dropping `-slim` was required to use `curl`?

Reply:
honestly i didn't check. I just assumed that if `slim` is to have meaning, it wouldn't include things like `curl`... but i was wrong.

- node:18.15.0 is a descendant of bullseye-slim https://hub.docker.com/layers/library/node/18.15.0-slim/images/sha256-e9ea1b9c6ed95b612a2766b7d9399ba35a8cf8068558ca631e08e19c6d6db92d?context=explore
- and the node bullseye-slim 18 dockerfile explicitly installs `curl`! https://github.com/nodejs/docker-node/blob/3c4fa6daf06a4786d202f2f610351837806a0380/18/bullseye-slim/Dockerfile#L20

Reverting to slim!